src/ADAL.PCL/TokenCache.cs

```diff
@@ -344,13 +344,16 @@ internal AuthenticationResultEx LoadFromCache(CacheQueryData cacheQueryData, Cal
         {
             lock (cacheLock)
             {
+                Telemetry.GetInstance().StartEvent(callState.RequestId, EventConstants.CacheLookUp);
+                CacheEvent cacheEvent = new CacheEvent();
                 PlatformPlugin.Logger.Verbose(callState, "Looking up cache for a token...");
 
                 AuthenticationResultEx resultEx = null;
                 KeyValuePair<TokenCacheKey, AuthenticationResultEx>? kvp = this.LoadSingleItemFromCache(cacheQueryData, callState);
 
                 if (kvp.HasValue)
                 {
+                    cacheEvent.SetEvent(EventConstants.TokenFound, true);
```

Comment:
Yes This exists in the telemetry spec in the CacheEvent.Added a tag for AT.

---
In reply to: [86877593](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/494#discussion_r86877593) [](ancestors = 86877593)